tests/core/data_model/test_dataobject.py

```diff
@@ -727,3 +740,33 @@ class Foo(DataObjectBase):
         descriptor.fields_by_name["float64"].type
         == _descriptor.FieldDescriptor.TYPE_DOUBLE
     )
+
+
+@pytest.mark.parametrize("run_num", range(500))
+def test_dataobject_jsondict(temp_dpool, run_num):
```

Comment:
Since this test is really fast, I'd like to keep the repetition just in case we haven't actually worked through all the issues. If there's any chance it will fail, I want to catch it early! That said, we could definitely use a comment about this.

Reply:
each run is < 0.005s so runtime-wise I think it's reasonably okay to keep, but I'd be fine removing now that we understand the segfault